Header header

uint8 fw_version

uint8 fw_revision

uint64 cpu_board_serno

uint16 system_configuration

uint8 beam_count

# Set of sensors available on the device
sonia_msgs/Sensors available_sensors
